Item.cs

```diff
@@ -26,25 +25,25 @@ public enum TypeItem
 
```

Comment:
remove this:  private static int _nextID = 1;
instead use GUID as an initial ID: Guid g = Guid.NewGuid();